theme/templates/resource-landing-page/modals.html

```diff
@@ -867,8 +867,9 @@ <h4 class="modal-title" id="add-creator"> Add Author </h4>
             </div>
 
             {#  Organization  #}
-            <!--<div data-hs-user-type="organization" style="display:none">
-                <div class="modal-body">
+            <div data-hs-user-type="organization" style="display:none">
+                {# This code block should be uncommented after the auto complete for Existing HS Groups is working #}
```

Comment:
Why the autocomplete for existing HS group is not working here? It seems to work on access control modal dialog.